### PR TITLE
Added 2 cmd. Done #25,#28

### PR DESCRIPTION
**Pull request by knight-ryu12**

### CODE REVIEW

**Comment by knight-ryu12** [comment]
Thank god Fred :+1:

File: FredBoat/src/main/java/fredboat/commandmeta/abs/ICommand.java [hunk 1]
```diff
@@ -25,13 +25,14 @@
...
+import com.mashape.unirest.http.exceptions.UnirestException;
 import net.dv8tion.jda.core.entities.Guild;
 import net.dv8tion.jda.core.entities.Member;
 import net.dv8tion.jda.core.entities.Message;
 import net.dv8tion.jda.core.entities.TextChannel;
 
 public interface ICommand {
 
-    public abstract void onInvoke(Guild guild, TextChannel channel, Member invoker, Message message, String[] args);
+    public abstract void onInvoke(Guild guild, TextChannel channel, Member invoker, Message message, String[] args) throws UnirestException;
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/commandmeta/abs/ICommand.java]: This change affects all commands. Please handle this with a try/catch in the ;;invite command

File: FredBoat/src/main/java/fredboat/command/util/InviteCommand.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ */
+public class InviteCommand extends Command {
+    @Override
+    public void onInvoke(Guild guild, TextChannel channel, Member invoker, Message message, String[] args)  {
+        try {
+            String str = "https://discordapp.com/oauth2/authorize?&client_id=" + DiscordUtil.getApplicationInfo(invoker.getJDA().getToken().substring(4)).getString("id") + "&scope=bot";
+            String send = MessageFormat.format(I18n.get(guild).getString("invite"),DiscordUtil.getApplicationInfo(message.getJDA().getToken().substring(4)).getString("name"));
+            channel.sendMessage(send + "\n" + str).queue();
+        } catch (UnirestException e) {
+            e.printStackTrace();
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/InviteCommand.java]: This is not a proper way to log an Exception nor does it alert the user that something has gone wrong. Please use this:
```java
throw new RuntimeException(e);
```

**Comment by Frederikam** [comment]
Great, now you just need to add the strings to the translation project:
https://github.com/Frederikam/FredBoat-i13n/blob/l10n_master/en_US.properties

**Pull request merged**